apps/platform/src/components/ui/line-tab.tsx

```diff
@@ -46,14 +49,17 @@ function Tab({
       }}
       type="button"
     >
-      <span className="relative z-10">{text}</span>
+      <span className="relative z-10 flex items-center gap-2">
+        {icon && <span className={'h-4 w-4'}>{icon}</span>}
+        {text}
+      </span>
       {selected ? (
         <motion.div
           className="absolute left-0 top-0 flex size-full items-end justify-center"
           layoutId={`${customID}linetab`}
           transition={{ type: 'spring', duration: 0.4, bounce: 0, delay: 0.1 }}
         >
-          <span className="z-0 h-[3px] w-[60%] rounded-t-full bg-white" />
+          <span className="z-0 h-[3px] w-[70%] rounded-t-full bg-white" />
```

Comment:
Maybe this is the section @Sawan was referring to. If we get rid of the partial width, the bottom line would cover the entire tab.